### PR TITLE
Prey.Prey version 1.9.10

### PR DESCRIPTION
**Pull request by techflashYT**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16381)

### CODE REVIEW

**Comment by sanikdah** [comment]
Quick note here for when the mods come to look at this, when I installed it I was confused on what had happened because it wasn't in the start menu and nothing came up when searching in the search box, but for some reason it installed itself into C:\Windows\Prey\, I'm not sure if that is intentional or not, just a heads up if you find that confusing when looking over logs or whatever.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
​

**Comment by sanikdah** [comment]
> I cannot find where this application is installed to; it's not showing up in control panel, start menu's app list, desktop shortcuts, program files, etc.

Did you even read my comment -_-

**Comment by ItzLevvie** [comment]
Sorry, I built a script that validates and installs a particular manifest by just using the pull request number; and make suggestions based on what's shown in Control Panel, so I don't usually check the comment section all that much, but I'll make sure to check it next time 👍 .

File: manifests/p/Prey/Prey/1.9.10/Prey.Prey.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Prey.Prey
+PackageVersion: 1.9.10
+PackageLocale: en-US
+Publisher: Prey
```
Comment (ItzLevvie) [inline manifests/p/Prey/Prey/1.9.10/Prey.Prey.locale.en-US.yaml:6]:
```suggestion
Publisher: Prey, Inc.
```

**Comment by sanikdah** [comment]
> Sorry, I built a script that validates and installs a particular manifest by just using the pull request number; and make suggestions based on what's shown in Control Panel, so I don't usually check the comment section all that much, but I'll make sure to check it next time 👍 .

It's OK, I understand :)

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**